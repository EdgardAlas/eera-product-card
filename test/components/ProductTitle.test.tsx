import React from 'react';
import renderer from 'react-test-renderer';
import ProductCard, { ProductTile } from '../../src/components';
import { product1 } from '../data/products';

describe('ProductTitle', () => {
  test('debe de mostrar el componente con el titulo personalizado', () => {
    const wrapper = renderer.create(
      <ProductTile title="test" className="cutom-class" />
    );
    expect(wrapper.toJSON()).toMatchSnapshot();
  });

  test('debe de mostrar el componente con el titulo personalizado', () => {
    const wrapper = renderer.create(
      <ProductCard product={product1}>
        {() => <ProductTile title="test" className="cutom-class" />}
      </ProductCard>
    );
    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
