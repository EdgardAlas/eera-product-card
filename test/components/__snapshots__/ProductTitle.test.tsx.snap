// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductTitle debe de mostrar el componente con el titulo personalizado 1`] = `
<span
  className="productDescription cutom-class"
>
  test
</span>
`;

exports[`ProductTitle debe de mostrar el componente con el titulo personalizado 2`] = `
<div
  className="productCard "
>
  <span
    className="productDescription cutom-class"
  >
    test
  </span>
</div>
`;
